runtime/laos/src/benchmarks.rs

```diff
@@ -15,19 +15,19 @@
 // along with LAOS.  If not, see <http://www.gnu.org/licenses/>.
 
 frame_benchmarking::define_benchmarks!(
-	[frame_system, SystemBench::<Runtime>]
+	// [frame_system, SystemBench::<Runtime>]
```

Comment:
On this comments, can we put this comment from the previous version explaining why we don't use this specific pallets:

``` 
 # It fails but others parachain as Moonbeam does not have this
  "frame_system" 
  # We use default weights from polkadot-sdk as benchmarking functions does not support existential deposit zero
  "pallet_balances" ```